Sources/core/objects/Tables.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "Movable.h"
+
+class Table: public IMovable
+{
+public:
+    DECLARE_SAVED(Table, IMovable);
+    DECLARE_GET_TYPE_ITEM(Table);
+    Table(size_t id);
+    virtual void AttackBy(IdPtr<Item> item) override;
+    virtual void Delete() override;
+    void NotifyNeighborhood(bool is_in_existence);
+    void UpdateSprite(size_t ignored_table);
+    virtual void AfterWorldCreation() override;
+    int CheckTable(IdPtr<IOnMapBase> container, size_t ignored_table);
+protected:
```

Comment:
`CheckTable` is used only inside `Table` class and does not use any class members -> it should be static and private.